pkg/nmi/server/server_test.go

```diff
@@ -31,7 +31,7 @@ func TestGetTokenForMatchingIDBySP(t *testing.T) {
 		Spec: aadpodid.AzureIdentitySpec{
 			Type:           aadpodid.ServicePrincipal,
 			TenantID:       "tid",
-			ClientID:       "cid",
+			ClientID:       "aabc0000-a83v-9h4m-000j-2c0a66b0c1f9",
```

Comment:
can the test assert the finally generated string ?

Reply:
We can't check the log to assert, but I think I can move it to a util function and unit test that.